src/main/java/com/uid2/shared/audit/Audit.java

```diff
@@ -81,11 +81,12 @@ public Builder(int status, String source, String method, String endpoint, String
                 this.method = method;
```

Comment:
I think it makes more sense for the Builder constructor to take requestId (or awsTraceId for even more clarity), uidTraceId

and in the constructor: if uidTraceId is null, set it to awsTraceId